.ci/error-prone-check.groovy

```diff
@@ -82,7 +83,14 @@ private static int checkErrorProneReport(String profile, String flag) {
  */
 private static List<String> getErrorProneErrors(String profile) {
     final List<String> errorProneErrors = [] as ArrayList
-    final String command = "mvn -e --no-transfer-progress clean" +
+    final String maven
+    if ("\\".equals(SEPARATOR)) {
+        maven = "mvn.cmd"
```

Comment:
Why?
https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/diff.groovy#L219

Reply:
That is a user-defined method:
```groovy
def executeCmd(cmd, dir = new File("").absoluteFile) {
    println "Running command: ${cmd}"
    def osSpecificCmd = getOsSpecificCmd(cmd)
    def proc = osSpecificCmd.execute(null, dir)
    proc.consumeProcessOutput(System.out, System.err)
    proc.waitFor()
    if (proc.exitValue() != 0) {
        throw new GroovyRuntimeException("Error: ${proc.err.text}!")
    }
}
```

Reply:
Yes, but it doesn't need to swap from running `mvn` to `mvn.cmd`. Also, I executed this on Windows and it ran fine without the `.cmd`.

Reply:
I got `java.io.IOException: Cannot run program "mvn": CreateProcess error=2, The system cannot find the file specified`, are the current changes fine?

Reply:
I am good with the new changes and am resolving this.